uploading/models/letterboxd_tables/ratings.js

```diff
@@ -1,10 +1,45 @@
 const db = require("../../../database/dbConfig.js");
 
-module.exports = { addRating };
+module.exports = { 
+  addRating,
+  getRatingById
+  // getRatings
```

Comment:
This is for the next RC?

Reply:
It will be necessary for future releases yes, but since I changed the initial models to not return I needed a way to retrieve the added files for testing.